dotnet-directory

```diff
@@ -66,6 +66,24 @@ print_framework() {
     exit 0
 }
 
+print_aspnet_framework() {
```

Comment:
nit: may be avoid the duplication and pass `Microsoft.AspNetCore.App` / `Microsoft.NETCore.App` as an argument to print_framework.